scripts/app.js

```diff
@@ -2,23 +2,64 @@
 
 var projects = [];
 
-function Projects (title, category, author, authorUrl, body) {
-  // done: Use the JS object passed in to complete this constructor function:
-  // Save ALL the properties of `rawDataObj` into `this`
-  //test
-  this.title = title;
-  this.category = category;
-  this.author = author;
-  this.authorUrl = authorUrl;
-  this.body = body;
-}
-//function to put projects in a 3x2 table
+//function to put projects in a 3x1 table
 function Portfolio(image, title, ghPagesUrl) {
```

Comment:
You are passing 'image' in the parameter but assigning 'img' to 'this.img'... You will also need to change the logic a bit to get the constructor to work with portfolioData.js